app/views/directives/process-template-dialog/process-template-select.html

```diff
@@ -3,6 +3,7 @@
     <div class="select-project-for-template">
       <h2>Select from Project</h2>
       <ui-select
+        ng-disabled="$ctrl.numTemplateProjects < 2"
```

Comment:
Instead of a disabled select, is it better to just have static text saying the project name here? That's what we've done elsewhere in the console (e.g. container picker on the view logs tab).

If the select is disabled, I would wonder why. I don't think it's obvious because you only have one project.

Reply:
I thought the same, but @beanh66 thought it better to be consistent with the select-project component.

Reply:
@spadgett I was trying to be consistent with what we did for the catalog wizard @dtaylor113 worked on. When there is only 1 project there, the dropdown is disabled. @cshinn any thoughts?